code/modules/mining/machine_stacking.dm

```diff
@@ -68,7 +68,7 @@
 	var/input_dir = null
 	var/output_dir = null
 	var/list/stack_storage
-	var/stack_amt = 50; // Amount to stack before releassing
+	var/stack_amt = 120; // Amount to stack before releassing
```

Comment:
No real need for that `;`. I get that it was there before, but it's still useless. Remove it.

Reply:
I'm too used to looking at C# code that I probably didn't even notice it. I'll get rid of that.